src/lib/repeat.ts

```diff
@@ -40,14 +43,16 @@ export function repeat<T>(items: Iterable<T>, keyFnOrTemplate: KeyFn<T>|ItemTemp
     let state = stateCache.get(part);
     if (state === undefined) {
       state = {
-        keyMap: keyFn && new Map(),
-        parts: [],
+        keyMap : keyFn && new Map(),
```

Comment:
This is interesting, don't think I have seen anyone use this style before..

Reply:
same, would prefer no space between the colon

Reply:
Same, but every formatter I've tried - prettier, clang and tsfmt - have their warts. We can file bugs on clang.

Reply:
fixed with a .clang-format